src/EdgeOrder/test/Az.EdgeOrder-TestResults.xml

```diff
@@ -0,0 +1,199 @@
+﻿<?xml version="1.0" encoding="utf-8" standalone="no"?>
```

Comment:
Test result is not needed to be checked in.
You should check in the .gitignore file.